wrapper/Widgets.cpp

```diff
@@ -198,6 +198,12 @@ IggBool iggCollapsingHeader(const char *label, int flags)
    return ImGui::CollapsingHeader(label, flags) ? 1 : 0;
 }
 
+
+IggBool iggTreeNodeID(char const *id, char const *label, int flags)
+{
+   return ImGui::TreeNodeEx(id, flags, label) ? 1 : 0;
```

Comment:
small caveat - the `label` is actually a format string in this call. Unfortunately, there's no version of `TreeNode*` that allows a specific ID and a plain label string